src/App.tsx

```diff
@@ -204,6 +207,16 @@ function App() {
       ],
       githubUsername: "narze",
     },
+    {
+      desc: "useLongerState - it will return such a longer hook than original useState. Hahaha",
+      examples: [
+        {
+          code: "const [countLongerState , setCountLongerState] = useLongerState[0]",
+          value: `Current count is ${countLongerState}`
```

Comment:
```suggestion
          value: `Current count is ${useLongerState()[0]}`
```

Reply:
Thank you, very much krub.
I was never though out like this before.